README.md

```diff
@@ -714,6 +715,21 @@ convert dbprotect2hdf         Translate a DBProtect report in "Check Results
       $ saf convert dbprotect2hdf -i check_results_details_report.xml -o output-hdf-name.json
 ```
 
+[top](#convert-other-formats-to-hdf)
+##### Dependency-Track to HDF
+```
+convert dependency_track2hdf       Translate a Dependency-Track results FPF file into
+                                              a Heimdall Data Format JSON file
+
+OPTIONS
```

Comment:
can we note which ones are required flags and add the -h help option to make it consistent with the rest? also I think the indentation got screwed up on this one